mne/viz/evoked.py

```diff
@@ -200,8 +200,62 @@ def _plot_evoked(evoked, picks, exclude, unit, show, ylim, proj, xlim, hline,
         (x axis: time, y axis: channel). In 'image' mode, the plot is not
         interactive.
     """
-    time_unit, times = _check_time_unit(time_unit, evoked.times)
     import matplotlib.pyplot as plt
+
+    # For evoked.plot_image ...
+    # First input checks for group_by and axes if any of them is not None.
+    # Either both must be dicts, or neither.
+    # If the former, the two dicts provide picks and axes to plot them to.
+    # Then, we call this function recursively for each entry in `group_by`.
+    if plot_type == "image" and isinstance(group_by, dict):
+        if axes is None:
+            axes = dict()
+            for sel in group_by:
+                plt.figure()
+                axes[sel] = plt.axes()
+        elif isinstance(axes, dict):
```

Comment:
Why not raise the error with `elif not isinstance(axes, dict)`?

Reply:
...